pages/common/git-authors.md

```diff
@@ -0,0 +1,17 @@
+# git authors
+
+> Display log or list of committers of a Git repository. Everything except `--list` will create a new file called `AUTHORS`.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-authors>.
+
+- Display a log of committers in the default editor, reading from the `AUTHORS` file:
+
+`git authors`
+
+- Print a full list of committers:
```

Comment:
```suggestion
- Display a full list of committers:
```

Reply:
> Also the the usage of display and print is used since the display to me indicates that it isn't a firm output, rather something that can be scrolled through, while the print is used to indicate that it just dumps the entire lot, as if it was concatenated using cat

I kind of have to agree with @CleanMachine1 here

Reply:
It is print since it dumps it, display to me indicates more of a elegant way of displaying.
the display indicates that it is opened in a text editor

Reply:
> > Also the the usage of display and print is used since the display to me indicates that it isn't a firm output, rather something that can be scrolled through, while the print is used to indicate that it just dumps the entire lot, as if it was concatenated using cat
> 
> I kind of have to agree with @CleanMachine1 here

couldn't find my reply! Was looking for it